packages/designmanual/stories/pages/SearchResultTypeExample.jsx

```diff
@@ -37,7 +46,7 @@ const responseDataSource = [
 const searchResults = [...responseDataSource, subjectDataSource];
 const initialTypeFilter = {};
 searchResults.forEach(item => {
-  const pageSize = item.type === contentTypes.SUBJECT ? 2 : 4;
+  const pageSize = item.type === contentTypes.SUBJECT ? 10 : 4;
```

Comment:
Skal det vises 10 fag når vi har rader på 2 eller 3? Er det ikke bedre med 6 eller 12?

Reply:
Dette er tatt bort. Det er ikke paginering på fag og forklaringer, alle skal vises.
Holder på å refaktorere heie eksempelkoden, så for deg er det nok best om du bare ser på hvordan data sendes inn til komponentene og ikke håndteringen av state.